src/helper.h

```diff
@@ -306,14 +298,14 @@ class helper_stateful_server : public HelperServerBase
     /// close handler to handle exited server processes
     static void HelperServerClosed(helper_stateful_server *srv);
 
-    statefulhelper *parent;
+    statefulhelper *parent = nullptr;
 
     // Reservations temporary lock the server for an exclusive "client" use. The
     // client keeps the reservation ID as a proof of her reservation. If a
     // reservation expires, and the server is reserved for another client, then
     // the reservation ID presented by the late client will not match ours.
     Helper::ReservationId reservationId; ///< "confirmation ID" of the last
-    time_t reservationStart; ///< when the last `reservation` was made
+    time_t reservationStart = 0;         ///< when the last "reservation" was made
```

Comment:
I do not know why that word was quoted as if it is a class member before, but I do not think we need to quote it at all. Other comments use "reservation" without quotes already.

```suggestion
    time_t reservationStart = 0; ///< when the last reservation was made
```